features/net/FEATURE_IPV4/TESTS/mbedmicro-net/udp_echo_client/main.cpp

```diff
@@ -66,6 +66,8 @@ int main() {
             result = false;
```

Comment:
@jeromecoutant Oops, not sure how I missed that, thanks for pointing that out. You're right, this line should stay in then.